src/FixedDataTableRowBuffer.js

```diff
@@ -39,11 +40,13 @@ class FixedDataTableRowBuffer {
     this._viewportRowsBegin = 0;
     this._viewportRowsEnd = 0;
     this._maxVisibleRowCount = Math.ceil(viewportHeight / defaultRowHeight) + 1;
-    this._bufferRowsCount = clamp(
-      Math.floor(this._maxVisibleRowCount/2),
-      MIN_BUFFER_ROWS,
-      MAX_BUFFER_ROWS
-    );
+    this._bufferRowsCount = bufferRowCount !== undefined
```

Comment:
Ahh no worries I'm chilling.  Just let me know what you two would prefer when you've nailed it.